go.mod

```diff
@@ -1,19 +1,19 @@
 module github.com/viam-modules/queue-estimator
 
-go 1.23
+go 1.23.0
 
 require (
 	github.com/pkg/errors v0.9.1
-	go.viam.com/rdk v0.50.0
-	go.viam.com/test v1.2.3
-	go.viam.com/utils v0.1.113
+	go.viam.com/rdk v0.70.0
```

Comment:
This is the only line in this file I changed manually. All the rest of the changes (including line 3) came from running `go mod tidy`. :shrug: